net/shadowsocks/src/etc/inc/plugins.inc.d/shadowsocks.inc

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+    Copyright (C) 2018 Michael Muenz
+    All rights reserved.
```

Comment:
I believe this is missing the Deciso B.V. Copyright

Reply:
no, it's a simple file. in fact, especially for the mutable controllers now the deciso copyright should not be pulled in (there is no code associated, see below)

Reply:
so no copyright at all? I'll be glad to remove anything since I have no idea of this thing :)